backend/pkg/logger/logger.go

```diff
@@ -13,33 +13,40 @@ type Logger struct {
 	// An atomic boolean is used in order to use CompareAndSwap in the Start and Stop methods
 	running        *atomic.Bool
 	subloggersLock *sync.RWMutex
-	subloggers     map[abstraction.LoggerName]abstraction.Logger
+	// The keys are only the loggers selected at the start of the log
+	keys map[abstraction.LoggerName]abstraction.Logger
```

Comment:
keep the old name and just say that because we don´t use the start and stop keys now in the start and stop methods we start all the loggers